### PR TITLE
NOTIF-603 Polish the orgId migration in engine

### PR DESCRIPTION
**Pull request by gwenneg**
This PR adds the `orgId` in most places where it was missing in `notifications-engine`.

After that, `CamelTypeProcessor` should be the only class where the `orgId` is missing. As explained below, it will be updated separately.

### CODE REVIEW

File: engine/src/main/java/com/redhat/cloud/notifications/processors/camel/CamelTypeProcessor.java [hunk 1]
```diff
@@ -41,6 +41,7 @@
 import static com.redhat.cloud.notifications.models.NotificationHistory.getHistoryStub;
 import static java.nio.charset.StandardCharsets.UTF_8;
 
+// TODO NOTIF-603 Migrate to orgId (not started in this class)
 @ApplicationScoped
 public class CamelTypeProcessor implements EndpointTypeProcessor {
```
Comment (gwenneg) [inline engine/src/main/java/com/redhat/cloud/notifications/processors/camel/CamelTypeProcessor.java:46]: This will be done in a separate PR. It requires changes in our integration with RHOSE.

Comment (gwenneg) [reply engine/src/main/java/com/redhat/cloud/notifications/processors/camel/CamelTypeProcessor.java:46]: https://issues.redhat.com/browse/NOTIF-706

File: engine/src/test/java/com/redhat/cloud/notifications/events/LifecycleITest.java [hunk 2]
```diff
@@ -461,7 +463,7 @@ void addEventTypeBehavior(UUID eventTypeId, UUID behaviorGroupId) {
 
     @Transactional
     void clearEventTypeBehaviors(EventType eventType) {
-        entityManager.createQuery("DELETE EventTypeBehavior WHERE eventType = :eventType")
+        entityManager.createQuery("DELETE FROM EventTypeBehavior WHERE eventType = :eventType")
```
Comment (josejulio) [inline engine/src/test/java/com/redhat/cloud/notifications/events/LifecycleITest.java:466]: Was this wrong before? Or is just better (style-wise) to use the `FROM` ?

Comment (gwenneg) [reply engine/src/test/java/com/redhat/cloud/notifications/events/LifecycleITest.java:466]: I think Hibernate automatically added the missing `FROM` but IntelliJ complained about this and I wanted him to be happy 😄
